spec/allure_config.rb

```diff
@@ -13,12 +13,12 @@
     ruby_version: RUBY_VERSION,
     host_runner_image: ENV['HOST_RUNNER_IMAGE'],
   }.compact
-  meterpreter_name = ENV['METERPRETER']
-  meterpreter_runtime_version = ENV['METERPRETER_RUNTIME_VERSION']
-  if meterpreter_name.present?
-    environment_properties[:meterpreter_name] = meterpreter_name
-    if meterpreter_runtime_version.present?
-      environment_properties[:meterpreter_runtime_version] = "#{meterpreter_name}#{meterpreter_runtime_version}"
+  session_name = ENV['SESSION']
```

Comment:
Do you need to update the README docs, as well as the github yml file that sets these environment variables? 👀

Reply:
Got those updated. Cheers 👍